scylladb/sphinx-scylladb-theme
# Style tables with headings

Closes https://github.com/scylladb/sphinx-scylladb-theme/issues/298

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/examples/tables/ with your favorite browser. The table with a heading should look like:

![image](https://user-images.githubusercontent.com/9107969/148230034-951fc534-b03f-4b97-88cd-e6321ab7d2c1.png)
